source/_includes/asides/docs_navigation.html

```diff
@@ -40,16 +40,8 @@ <h1 class="title delta">Topics</h1>
         </ul>
       </li>
       <li>
-        <b>{% active_link /docs/assist/ Assist %}</b>
+        <b>{% active_link /voice/ Assist %}</b>
         <ul>
-          <li>{% active_link /docs/assist/android/ Assist for Android %}</li>
```

Comment:
If you change URLs, you need to update the `redirect` file to make sure all old URLs keep working.